src/api/types.ts

```diff
@@ -182,4 +182,23 @@ export type SearchHitLine = {
   content: string
 }
 
+export interface ProgramExportImportInfo {
+  program_library: string, 
+  program_name: string, 
+  object_type: string, 
+  symbol_name: string, 
+  symbol_usage: string,
+  argument_optimization: string,
+  data_item_size: number
+}
+
+export interface ModuleExport {
+  module_library: string, 
+  module_name: string, 
+  module_attr: string, 
+  symbol_name: string, 
+  symbol_type: string,
+  argument_optimization: string,
+}
```

Comment:
Prefer camel case for variable names, to keep it consistent with the others.
```suggestion
export interface ProgramExportImportInfo {
  programLibrary: string
  programName: string
  objectType: string
  symbolName: string
  symbolUsage: string
  argumentOptimization: string
  dataItemSize: number
}

export interface ModuleExport {
  moduleLibrary: string
  moduleName: string
  moduleAttr: string
  symbolName: string
  symbolType: string
  argumentOptimization: string
}
```